GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1938,6 +1946,7 @@ private void SetGitModule(object sender, GitModuleEventArgs e)
 
                 RevisionGrid.IndexWatcher.Reset();
                 MainSplitContainer.Visible = false;
+                RegisterPlugins();
```

Comment:
We call this in the other branch too, should this be moved either before or after the if..else...?

Reply:
@RussKie 
Things seem to still wrok after this diff:
![image](https://user-images.githubusercontent.com/14078392/96671704-73609700-1330-11eb-9edb-e4920a6f777e.png)

Is this a change you would like to be made?

Reply:
Yep, thanks

Reply:
@RussKie done!